api/controllers/getJSON.js

```diff
@@ -1,8 +1,8 @@
-const prepVerse = (row, includeMeta = false) => {
+const prepVerse = (row, includeMeta = false, liveSearch = 0) => {
```

Comment:
nit: would prefer defaulting to `false`

Reply:
yeah but the values are integers. If I did false here, possible values become 1,0,false